keras_core/ops/random_test.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from absl.testing import parameterized
+
+from keras_core import ops
+from keras_core import testing
+
+
+class RandomTest(testing.TestCase, parameterized.TestCase):
+    def test_shuffle(self):
+        x = np.arange(100).reshape(10, 10)
+
+        # Test axis=0
+        expected = ops.random.shuffle(x)
+
+        self.assertFalse(
+            np.all(x == ops.convert_to_numpy(expected))
+        )  # seems unlikely!
```

Comment:
Unlikely but nevertheless the test will be flaky some % of the time unless it is seeded. I recommend using a seed.

Reply:
Done. `seed=0` has been added.